src/routes/components/collapsible/+page.md

```diff
@@ -0,0 +1,125 @@
+<script lang="ts">
+    import Collapsible from "$lib/components/Collapsible.svelte";
+    let toggleButton: HTMLElement;
+    let toggleContentF: function;
```

Comment:
`: Function` or `: () => void`

Reply:
but you don't need necessary all these variables here, the showcase case use inline constant and dummy function